#41 fixing effectless --preview-text parameter for file output

myselfhimself opened the pull request:
Fixing #41 fully

sdushantha commented:
Good catch! I did not notice this.
Could you explain how your changes fixes this? From looking at your changes, all you did was move some code into a function

myselfhimself commented:
Hello
the code did not have to be moved into a function, but I moved it into a
function to make it "cleaner".
The fix consists in running the multiple variables fallback lines not just
at main() time, but before the 2 preview commands code which get run before
the main() itself.
So I moved this variables fallback chunk outside of the main to be before
the 2 preview commands and main itself, so it serves 3 cases instead of
just 1.
Have a nice day

st 24. 6. 2020 v 12:22 odesílatel Siddharth Dushantha <
notifications@github.com> napsal:

> Good catch! I did not notice this.
> Could you explain how your changes fixes this? From looking at your
> changes, all you did was move some code into a function
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sdushantha/fontpreview/pull/42#issuecomment-648734963>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJU5QX2GKFQUCKHHOYVJG3RYHHX7ANCNFSM4OGEN7KQ>
> .
>
